demisto_sdk/commands/common/tools.py

```diff
@@ -1105,6 +1107,14 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
                     return FileType.INCIDENT_FIELD
                 if _id.startswith('indicator'):
                     return FileType.INDICATOR_FIELD
+                else:  # not an incident or an indicator
+                    if 'definitionId' in _dict and 'genericModuleId' in _dict:
+                        if 'cliName' in _dict:
+                            return FileType.GENERIC_FIELD
+                        elif 'color' in _dict:
+                            return FileType.GENERIC_TYPE
+                    elif 'definitions' in _dict and 'views' in _dict:
+                        return FileType.GENERIC_MODULE
```

Comment:
Move this above line 1103 (before field identification) - only modules will have views and definitions